netbox/netbox/tables/columns.py

```diff
@@ -285,7 +285,7 @@ def render(self, record, table, **kwargs):
         for idx, (action, attrs) in enumerate(self.actions.items()):
             permission = get_permission_for_model(model, attrs.permission)
             if attrs.permission is None or user.has_perm(permission):
-                url = reverse(get_viewname(model, action), kwargs={'pk': record.pk})
+                url = get_action_url(record, action=action, kwargs={'pk': record.pk})
```

Comment:
I'd keep this as passing `model` unless there's a need to change it.

```suggestion
                url = get_action_url(model, action=action, kwargs={'pk': record.pk})
```